modules/@apostrophecms/i18n/ui/apos/components/AposI18nLocalize.vue

```diff
@@ -713,18 +727,26 @@ export default {
     },
     // Get all related documents
     async getRelatedDocs(doc) {
+      const status = this.wizard.busy;
+      this.wizard.busy = true;
       const schema = apos.modules[doc.type].schema;
       const docs = getRelatedBySchema(doc, schema);
       if (!docs.length) {
         return [];
       }
-      const result = await apos.http.post(`${apos.doc.action}/editable?aposMode=draft`, {
-        body: {
-          ids: docs.map(doc => doc._id)
-        }
-      });
-      const filtered = docs.filter(doc => result.editable.includes(doc._id));
-      return filtered;
+      try {
+        const result = await apos.http.post(`${apos.doc.action}/editable?aposMode=draft`, {
+          body: {
+            ids: docs.map(doc => doc._id)
+          }
+        });
+        const filtered = docs.filter(doc => result.editable.includes(doc._id));
+        return filtered;
+      } catch (e) {
+        //
```

Comment:
You don't have to have a catch block just to have a finally block. Since nothing useful is happening in this catch block it is better practice to let it throwwww to the calling function. You can still have a finally block.